bin/new_mashfile.sh

```diff
@@ -72,11 +72,12 @@ case $SERIES in
          ;;
 esac
 
-# in OSG 23+, "contrib" and "empty" make it into the "$branch"; in previous, it's part of "$REPO"
-if [[ $REPO == development || \
-      $TAG  == *contrib* || \
-      $TAG  == *empty* ]]; then
-    if [[ $developer_key == "$auto_key" ]]; then
+# in OSG 23+, we want contrib, empty, testing, and release branches
```

Comment:
GitHub is not letting me make a suggestion. Here is the code block I'd use:
```
if [[ $REPO == development ]]; then
    KEYS=$auto_key
else
    KEYS=$developer_key
fi
```

Reply:
ah ok because we don't have any tags that reasonably need _both_ the dev and auto keys if we're not doing this for OSG 23 dev

Reply:
Fixed

Reply:
Exactly. If we're not having multiple keys for 23-development and we're not using the auto key for contrib or empty, and we don't have separate keys for not-23, then we can simplify a lot :)

Reply:
Actually we can probably get rid of the separate $auto_key and $developer_key vars in that case.